### PR TITLE
Add nix section in the README 

### PR DESCRIPTION
**Pull request by sevanspowell**
# Description

- Adds entries to the README to explain how to use the nix build.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Release Notes
+
+## v?
```
Comment (sevanspowell) [inline CHANGELOG.md:3]: Please let me know what the appropriate version is.

Comment (mirkoAlic) [reply CHANGELOG.md:3]: 3.0

File: README.md [hunk 2]
```diff
@@ -23,13 +23,15 @@ Depending on network you want to join you can use appropriate launcher, all can
 
 ### Building the client
 
-#### Prerequisites to build
+#### SBT
+
+##### Prerequisites to build
 
 - JDK 1.8 (download from [java.com](http://www.java.com))
```
Comment (mirkoAlic) [inline README.md:30]: btw, i think now is working ok with JDK 11

Comment (kapke) [reply README.md:30]: It is. Do we all want to jump into 11?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+### Introduce Nix expressions required to build mantis with Nix
+
+Use Sbtix to generate the Nix expressions we need to build mantis in a
+reproducible manner.
+
+#### built.sbt: "mantis" renamed to "mantis-core"
```
Comment (mirkoAlic) [inline CHANGELOG.md:10]: This was fixed here: https://github.com/input-output-hk/mantis/pull/671

**Comment by sevanspowell** [comment]
I've removed the CHANGELOG as most of it isn't relevant now (with the fix to the mantis naming). I'll let the dev team introduce one if they desire in future.

File: README.md (hunk 2 above)
Comment (mirkoAlic) [reply README.md:33]: who knows hehe..

**Review by mirkoAlic — APPROVED** [review]

**Pull request merged**